src/main/java/es/thalesalv/chatrpg/adapters/discord/listener/MessageListener.java

```diff
@@ -32,8 +35,9 @@ public class MessageListener {
 
     public void onMessageReceived(MessageReceivedEvent event) {
 
-        if (!event.getAuthor()
-                .isBot()) {
+        final User author = event.getAuthor();
+        final Message message = event.getMessage();
+        if (!author.isBot() && StringUtils.isNotBlank(message.getContentRaw())) {
```

Comment:
This confuses me. Discord doesn't accept empty sends AFAIK. I'm approving the PR, but this feels unnecessary to me.